sgl-kernel/benchmark/benchmark_lightning_attention_decode.py

```diff
@@ -0,0 +1,299 @@
+import itertools
```

Comment:
Add this file for micro benchmark with sgl and triton lightning_attention_decode kernel.